employees/views.py

```diff
@@ -219,7 +235,7 @@ def include_users_in_reports_dict(self, project: Project, year: int, month: int)
             queryset = self.get_queryset(project_pk=project.pk, author_pk=user.pk, year=year, month=month)
             user_reports_dict = query_as_dict(queryset)
             key = user.email
-            reports_dict[key] = user_reports_dict
+            reports_dict[key] = [user_reports_dict, decimal_to_hours_string(hours_per_month_counter(user_reports_dict))]
```

Comment:
This is fundamentally wrong because there should be no need to iterate through members and calculate hours for each separately, it should be also done in pull request, but lets not over complicate it now unless you want to refactor the whole method?